source/ut_utils.pks

```diff
@@ -11,10 +11,10 @@ create or replace package ut_utils is
     tr_failure - one or more asserts failed
     tr_error   - exception was raised
   */
-  tr_success     constant number(1) := 1; -- test passed
-  tr_failure     constant number(1) := 2; -- one or more asserts failed
-  tr_error       constant number(1) := 3; -- exception was raised
-	
+  tr_success          constant binary_integer := 1; -- test passed
+  tr_failure          constant binary_integer := 2; -- one or more asserts failed
+  tr_error            constant binary_integer := 3; -- exception was raised
```

Comment:
As all properties of objects are simple integers in every assignment operation implicit type cast would occur. I don't think this change in needed, or maybe just define constants as integers.